src/routes/explore/+page.svelte

```diff
@@ -1 +1,198 @@
+        </span>
+        <span class="font-semibold leading-none text-surface-50 text-base font-normal md:text-[1.1vw]">Unleash your inner Otaku: Explore anime wonders</span>
+    </section-headings>
+
+    <explore-options class="mt-7 md:mt-[2vw] flex flex-col gap-5 md:gap-0 md:flex-row md:items-end justify-between">
+        <search class="flex flex-col gap-1 md:gap-[0.35vw]">
+            <span class="leading-none text-surface-50 text-base md:text-[1vw]">Search Animes</span>
+            <div class="relative flex items-center">
+                <Search class="pointer-events-none absolute text-surface-50 ml-4 md:ml-[1vw] w-5 md:w-[1.25vw]" />
+                <input
+                    type="text"
+                    placeholder="Looking for specific anime? Start from here..."
+                    class="border-none bg-surface-400 leading-none placeholder:text-surface-50 focus:ring-0 w-full md:w-[50vw] rounded-lg md:rounded-[0.5vw] py-3 pl-14 md:py-[0.8vw] md:pl-[3vw] md:text-[1vw]"
+                />
+            </div>
+        </search>
+        <filter_page_tabs class="flex items-center justify-between">
+            {#each Object.entries(filter_pages_mapping) as page}
+                {@const page_key = page[0]}
+                {@const page_title = page[1].title}
+
+                {@const is_active = active_filter_page === page_key}
+
+                <button
+                    class="cursor-pointer text-base px-3 py-2 h-14 leading-tight md:h-auto rounded-lg md:px-[1.25vw] md:py-[0.9vw] md:rounded-[0.5vw] font-semibold md:text-[1vw] hover:text-white transition-colors leading-none"
+                    class:bg-surface-400={is_active}
+                    class:text-surface-50={!is_active}
+                    on:click={() => change_filter_page(page_key)}
+                >
+                    {page_title}
+                </button>
+            {/each}
+        </filter_page_tabs>
+    </explore-options>
+
+    <filter-options class="flex items-end gap-3 md:gap-0 justify-between mt-5 md:mt-0">
+        {#each Object.entries(filter_options_mapping) as option}
+            {@const title = option[1].title}
+            {@const value = option[1].value}
+            {@const klass = option[1].class}
+
+            <filter-component class={klass}>
+                <span class="leading-none text-surface-50 md:text-[1vw]">{title}</span>
+                <div class="relative flex items-center">
+                    <button class="btn absolute right-0 p-0 mr-3 md:mr-[1vw] w-4 md:w-[1.25vw]">
+                        <Chevron class="text-surface-50" />
+                    </button>
+                    <input
+                        type="text"
+                        class="border-none bg-surface-400 leading-none placeholder:text-surface-50 focus:ring-0 md:w-[11vw] w-full py-3 rounded-lg text-base md:rounded-[0.5vw] md:py-[0.8vw] md:pl-[1vw] md:text-[1vw]"
+                        {value}
+                    />
```

Comment:
This is the worst way to handle dropdowns. 

Check 
i. [`skeleton-autocomplete`](https://www.skeleton.dev/components/autocomplete)
ii. Disable the input on the input box